experimental/Geometry/Misc.jl

```diff
@@ -0,0 +1,202 @@
+import AbstractAlgebra.Ring
+import AbstractAlgebra.RingElement
+import Oscar: radical_membership
+
+
+module Misc
+using Oscar
+
+export add_variables, divides_power
+
+####################################################################
+#  
+#  Miscellaneous routines used in the above 
+#
+####################################################################
+
+
+function add_variables( R::T, new_vars::Vector{String} ) where{ T<:MPolyRing } 
+  k = base_ring(R)
+  old_vars = String.( symbols(R) )
+  n = length( old_vars )
+  vars = vcat( old_vars, new_vars )
```

Comment:
Elsewhere we accept both strings and symbols for variable names, so perhaps also here (and same for the other `add_variables` method below)?

```suggestion
add_variables(R::T, new_vars::Vector{String}) where {T<:MPolyRing} = add_variables(R, Symbol.(v))

function add_variables(R::T, new_vars::Vector{Symbol}) where {T<:MPolyRing}
  k = base_ring(R)
  old_vars = symbols(R)
  n = length( old_vars )
  vars = vcat( old_vars, new_vars )
```

Reply:
Ok, that makes sense. I'll adjust that.